Source/GSStoryboardTransform.m

```diff
@@ -0,0 +1,866 @@
+                  // Create action...
+                  NSXMLElement *action = [NSXMLElement elementWithName: @"action"];
+                  NSXMLNode *selector
+                    = [NSXMLNode attributeWithName: @"selector"
+                                       stringValue: @"doAction:"];
+                  NSXMLNode *target
+                    = [NSXMLNode attributeWithName: @"target"
+                                       stringValue: [[sbproxy attributeForName: @"id"] stringValue]];
+                  NSXMLNode *controller_ident
+                    = [NSXMLNode attributeWithName: @"id"
+                                       stringValue: uid]; 
+                  [action addAttribute: selector];
+                  [action addAttribute: target];
+                  [action addAttribute: controller_ident];
+                  [segue_parent addChild: action];
+                }
+              
+              // Create the segue...
+              NSStoryboardSegue *ss = [[NSStoryboardSegue alloc] initWithIdentifier: ident
+                                                                             source: src
+                                                                        destination: dst];
+              [ss _setKind: kind];
+              [ss _setRelationship: rel];
+              
+              // Add to maptable...
+              [mapTable setObject: ss
+                           forKey: ident];
+              
+            } // only process segue objects...
+        } // iterate over objects in each set of connections
+      END_FOR_IN(children);          
+    } // iterate over connection objs
+  END_FOR_IN(connectionsArray);
+
+  return mapTable;
+}
+
+- (void) processSegues: (NSXMLDocument *)xmlIn
+                 forId: (NSString *)identifier
+{
+  NSMutableString *uuidString = [NSMutableString string];
+  BOOL processed = [self isProcessedDocument: xmlIn
+                                        uuid: uuidString];
+  if (processed)
+    {
+      return; // don't processed the document again
+    }
+  else // Get the controller... there is only one per scene.
+    {
+      NSArray *array = [xmlIn nodesForXPath: @"//objects[1]"
+                                      error: NULL];
+      NSXMLElement *objects = [array objectAtIndex: 0]; // get the "objects" section
+      NSString *src = [self findControllerIdInDocument: xmlIn
+                                           withObjects: objects];
+      NSArray *connectionsArray = [xmlIn nodesForXPath: @"//connections"
+                                                 error: NULL];
+      NSMapTable *mapTable = [self processConnections: connectionsArray
+                                          withObjects: objects
+                                         controllerId: src];           
+      [_identifierToSegueMap setObject: mapTable
+                                forKey: identifier];                  
+      // Add to cache...
+      [_documentsMap setObject: mapTable
```

Comment:
I could not find any place where this is ever used. Could you please explain the idea?

Reply:
I've removed it.  After the reorganization, it is not needed.